### DISCUSSION TITLE
无法挂载PikPak

### DISCUSSION THREAD

**Discussion by linger6580** [AlistGo/alist#6224] [Q&A] [answered]
挂载PikPak后提示ERRO[2024-03-15 20:00:56] failed get enabled storages: Post "https://user.mypikpak.com/v1/auth/signin": read tcp 192.168.3.11:7561->8.219.103.183:443: wsarecv: An existing connection was forcibly closed by the remote host.

**Comment by linger6580** [comment, marked as answer]
在同样网络环境下的另外一台电脑就可以正常使用

**Reply by kaixinxiaowangzi666** [reply]
前面加+86 没

**Comment by linger6580** [comment]
> 前面加+86 没
邮箱登录的，不涉及+86的问题吧
